config/ci.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Model Configuration CI Testing Configuration",
+    "description": "Settings for Continuous Integration (CI) tests for model configurations repositories",
+    "type": "object",
+    "$defs": {
```

Comment:
This feature you showed me yesterday was very cool. Will be using it in future schema! (schemas? schemae?) 
Feel free to resolve this one :D